lambdas/lambdas.mk

```diff
@@ -1,4 +1,33 @@
 
+.PHONY: compile
+compile: check_env
+	PYTHONHASHSEED=0 python -m compileall -f -q --invalidation-mode checked-hash vendor vendor/azul
+
+# The compile target is used during packaging of lambdas. The target ensures
```

Comment:
I think the comment should precede the target. That's what we do elsewhere. The length of the comment makes that a bit awkward but I prefer the consistency.

Reply:
Sure. I was following the pattern here https://github.com/DataBiosphere/azul/blob/develop/common.mk#L89 (since I had originally put this target in common.mk). Should I move that comment or leave it as it is?

Reply:
That one comment is an exception because of the fact that .FORCE has to succeed the recipe. So I though it better to have the comment succeed it.